views/shared/header.html

```diff
@@ -1,23 +1,36 @@
-<div class="greeting">
-  {{ if .CurrentUser }}
-    <div class="greeting__message">Hi, {{ .CurrentUser.Email }} :)</div>
-    <div class="greeting__action">
-      <div class="greeting__link">
-        <a class="link link--primary" href="/dashboard">Dashboard</a>
-      </div>
-      <div class="greeting__link">
-        <a class="link link--primary" href="/user/sign_out">Sign Out</a>
-      </div>
+<nav class="navbar-panel">
+  <div class="navbar-panel__brand">
+    <a class="brand-logo" href="{{ urlfor `MainController.Index` }}" title="X42">
+      <img width="48" height="48" class="brand-logo__icon" src="/assets/images/favicon.svg" alt="X42">
+      <span class="brand-logo__name">X42</span>
+    </a>
+  </div>
+
+  <div class="navbar-panel__navigation">
+    <div class="user-panel">
+      {{ if .CurrentUser }}
+        <img width="40" height="40" class="user-panel__avatar" title="{{ .CurrentUser.Email }}" src="https://www.gravatar.com/avatar/{{ .CurrentUser.Email | hashEmail }}?d=retro&f=y" alt="avatar">
+        <ul class="user-panel__action">
```

Comment:
@Lahphim IMO it's a bit weird when there's no user signed in and all the navigation links are under `user-panel` 😆 
![Screen Shot 2564-02-19 at 17 22 51](https://user-images.githubusercontent.com/1772999/108492073-4a98e000-72d7-11eb-908a-089501d87521.png)

Can we move all the links out?

Reply:
updated https://github.com/Lahphim/go-crawler-challenge/pull/53/commits/054f45d16e63e515a882de350bd71a946e42aae3